.github/workflows/livestream-docker-image.yml

```diff
@@ -41,15 +41,15 @@ jobs:
                   images: ghcr.io/posthog/posthog/livestream
 
             - name: Set up Docker Buildx
-              uses: docker/setup-buildx-action@v2
+              uses: docker/setup-buildx-action@v3
```

Comment:
fly-by we can use Depot. we pay for it, it's way faster 😊 

(will dig out an example)

Reply:
quickstart here https://depot.dev/docs/container-builds/reference/github-actions